doc/user-guides/gateway-tls.md

```diff
@@ -0,0 +1,209 @@
+# Gateway TLS for Cluster Operators
+
+This user guide walks you through an example of how to configure TLS for all routes attached to an ingress gateway.
+
+<br/>
+
+## Requisites
+
+- [Docker](https://docker.io)
+
+### Setup
+
+This step uses tooling from the Kuadrant Operator component to create a containerized Kubernetes server locally using [Kind](https://kind.sigs.k8s.io),
+where it installs Istio, Kubernetes Gateway API, CertManager and Kuadrant itself.
+
+Clone the project:
+
+```shell
+git clone https://github.com/Kuadrant/kuadrant-operator && cd kuadrant-operator
+```
+
+Setup the environment:
+
+```shell
+make local-setup
+```
+
+Install metallb:
+```shell
+make install-metallb
+```
+
+Fetch the current kind networks subnet:
+```shell
+docker network inspect kind -f '{{ (index .IPAM.Config 0).Subnet }}'
+```
+Response:
+```shell
+"172.18.0.0/16"
+```
+
+Create IPAddressPool within kind network(Fetched by the command above) e.g. 172.18.200
+```shell
+kubectl -n metallb-system apply -f -<<EOF
+apiVersion: metallb.io/v1beta1
+kind: IPAddressPool
+metadata:
+  name: kuadrant-local
+spec:
+  addresses:
+  - 172.18.200.0/24
+---
+apiVersion: metallb.io/v1beta1
+kind: L2Advertisement
+metadata:
+  name: empty
+EOF
+```
```

Comment:
should we move this into a different file and set it as a pre-req? That way it wont be repeated and also when we have a quick start we can just update these docs to remove this setup piece

Reply:
Meh, i think these docs just listing all steps is better than docs linking to other docs expecting you to run all those steps before continuing steps in this doc. I'd rather just script this tbh, can just be done as part of `make install-metallb` initially and decide if you want to make it part of `make local-setup` later, wdyt?

Reply:
Yeah I think adding it to make install-metallb is a good call